interfaces/repositories.go

```diff
@@ -84,15 +94,20 @@ func (repo DbUserRepo) FindById(id int) (usecases.User, error) {
 }
 
 func (repo DbUserRepo) Count() (int, error) {
-	row, err := repo.dbHandler.Query("SELECT COUNT(*) FROM users")
+	//"SELECT COUNT(*)" returns number of row, not the last id in the database
+	row, err := repo.dbHandler.Query("SELECT id FROM users")
```

Comment:
This function gets the IDs to which the users are assigned when they are added to the database. It's possible that some users delete their account, making the number of rows different from the new IDs, so if I use ```COUNT(*)``` instead of ```id```, it's potentially wrong.